docs/siem/detections/api/lists/api-update-list-container.asciidoc

```diff
@@ -0,0 +1,80 @@
+[[lists-api-update-container]]
+=== Update list container
+
+Updates an existing list container.
+
+You can use `PUT` or `PATCH` methods to update list containers, where:
+
+* `PUT` replaces the original container and deletes fields that are not
+specified.
+* `PATCH` updates the specified fields.
+
+==== Request URL
+
+`PUT <kibana host>:<port>/api/lists`
+
+`PATCH <kibana host>:<port>/api/lists`
+
+==== Request body
+
+A JSON object with:
+
+* The `id` of the list container you want to update.
+* The fields you want to modify.
+
+IMPORTANT: If you call `PUT` to update a rule, all unspecified fields are
+deleted. You cannot modify the `id` and `type` fields.
+
+For `PATCH` calls, any of the fields can be modified, whereas for `PUT` calls,
+some fields are required. 
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description |Required (`PUT` calls)
+
+|`description` |String |Describes the list container. |Yes
+|`meta` |Object |Placeholder for metadata about the list container. |No
+|`name` |String |The list container's name. |Yes
+
+|==============================================
+
+
+===== Example request
+
+Updates the `name` field:
+
+[source,console]
+--------------------------------------------------
+PATCH api/lists
+{
+  "id": "internal-ip-excludes",
+  "name": "Internal IP addresses"
+}
+--------------------------------------------------
+// KIBANA
+
+==== Response code
+
+`200`:: 
+    Indicates a successful call.
+    
+==== Response payload
+
+The updated object, including the time it was updated.
+
+Example response:
+
+[source,json]
+--------------------------------------------------
+{
+  "created_at": "2020-07-08T07:26:55.672Z",
+  "created_by": "Threat Hunter",
+  "description": "Contains list items that exclude internal IP addresses from detection rules.",
+  "id": "internal-ip-excludes",
+  "name": "Internal IP addresses",
+  "tie_breaker_id": "527d559f-8777-49c0-859c-e24cbc329e7d",
+  "type": "ip",
+  "updated_at": "2020-07-08T08:46:55.470Z",
+  "updated_by": "Threat Hunter"
+}
```

Comment:
As mentioned above this will return the three new values of `version`, `_version`, and `immutable` but the `immutable` is not being used at the moment by any of us and always says false at the moment. Later if we implement immutable pre-packaged lists and list items then we would do something with it.